app/src/main/java/com/spendster/presentation/homeScreen/Profile/ProfileFragmentView.java

```diff
@@ -0,0 +1,6 @@
+package com.spendster.presentation.homeScreen.Profile;
+
+public interface ProfileFragmentView {
+    void setUsername(String username);
```

Comment:
@SimonMaha probably it's better to pass user object for this screen, not a separate fields. Also, please don't use "set" in prefix for methods.